proposals/wac-ucr/index.bs

```diff
@@ -76,7 +80,23 @@ individuals (i.e. Alice, Bob, Carol, etc.) are already
 [=authenticated agents=], and their corresponding [=identities=] are known
 by the [=resource controller=] when they are granted access.
 
-## Basic resource access ## {#uc-basic}
+Additionally, since the vast majority of these use cases involve a
+[=resource controller=] reviewing and modifying permissions as an
+[=authenticated agent=], the following requirements are not listed explicitly
+in every use case, but should be assumed when permissions are assigned. 
+They may appear in certain use cases (such as [[#basic-change-permissions]]) when
+relevant.
+
+<div class="assertion">
+
+* [[#req-read-permissions]]
+* [[#req-change-permissions]]
+* [[#req-collection-read-permissions]]
```

Comment:
This is "3.4.5. The system shall limit the ability to read the access permissions associated with a certain collection."
Same point as above.  There are many use cases where the ability to read the permissions of a collection is the obvious thing to do. So the statement here should be 
"The system shall allow read access to permissions associated with a certain collection to be limited."

Reply:
Adjusted in https://github.com/solid/authorization-panel/pull/152/commits/691e677fb30249fd00b4d3a547899a98a7713f51